examples/app_service/app-sample-blueprint.yaml

```diff
@@ -0,0 +1,116 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This Blueprint installs the nodecellar application
+  on an existing host.
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.4/types.yaml
+  - ../../plugin.yaml
+
+inputs:
+  resource_prefix:
+    default: cfy
+
+  resource_suffix:
+    default: 10
+
+  # Azure account information
+  subscription_id:
+    type: string
+    required: false
+
+  tenant_id:
+    type: string
+    required: false
+
+  client_id:
+    type: string
+    required: false
+
+  client_secret:
+    type: string
+    required: false
+
+  location:
+    type: string
+    required: true
+    default: eastus
+
+  password:
+    type: string
+
+  plan_name:
+    type: string
+    default: test-plan
+
+  resource_group_name:
+    type: string
+    default: test-group
+
+  web_app_name:
+    type: string
+    default: test-app
+
+dsl_definitions:
+  azure_config: &azure_config
+    subscription_id: { get_input: subscription_id }
+    tenant_id: { get_input: tenant_id }
+    client_id: { get_input: client_id }
+    client_secret: { get_input: client_secret }
+
+  plan_info: &plan_info
+    location: { get_input: location }
+    sku:
+      name: S1
+      capacity: 1
+      tier: Standard
+
+  app_config: &app_config
+    location: { get_input: location }
+    server_farm_id: sample-plan
+    site_config:
+      scm_type: LocalGit
+      python_version: 3.6
+
+  user: &user
+    name: sample-user
+    user_name: sample-user
+    publishing_password: STR0NG-test-Pa$$&
+    location: { get_input: location }
+
+node_templates:
+  user:
```

Comment:
What will someone do if they want to use two distinct users and apps in a single blueprint?

Reply:
you can define two of them in the blueprint then reference them in the nodes.
but, I think there is a limitation on on the users, seems like you can set one user only...
https://github.com/cloudify-incubator/cloudify-azure-plugin/pull/95/files#diff-b50a20f03cef68af08c85c266ddb99a4R34

it works on the webapp, you can define `app_config1` and `app_config2` for example, and set the values you want, at the end, those will be send as JSON to the SDK